### DISCUSSION TITLE
Implementing packet level- ways

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#15932] [Q&A] [answered]
I read through some links, could not understand the difference between

1. setting throttlingMethod as 'devtools' and setting
 throttling.requestLatencyMs=0 \ --throttling.downloadThroughputKbps=0 \ --throttling.uploadThroughputKbps=0 
--throttling.slowdownCPUmultiplier=1
and then using sitespeed to throttle the whole system.

vs

2. setting throttlingMethod as 'provided' and then using sitespeed to throttle the whole system.

To acheive packet level throttling, can i use either of the above? please clarify. What is the difference between the above two settings. 
Note: i am using lighthouse userflows in nodejs program for generating reports.

**Comment by connorjclark** [comment, marked as answer]
Packet level throttling is not provided by Lighthouse, so you would use "provided" and set it up yourself in your environment.

**Reply by SivasubramanianV** [reply]
@connorjclark thanks, what happens if i do the first configuration i mentioned, any difference you see between both?

**Reply by connorjclark** [reply]
That is not a well understood usage. If intention is to turn off all applied throttling, use "provided".

**Reply by SivasubramanianV** [reply]
@connorjclark what is the the throttle up, down and rtt that i should set using sitespeed. My application is mostly used in canada. Should it reflect the average network conditions of canada? Any idea where i can get that data from?
